static_src/actions/user_actions.js

```diff
@@ -221,6 +209,37 @@ const userActions = {
     return Promise.resolve(userInfo);
   },
 
+  fetchCurrentUserRole(userGuid) {
+    if (!userGuid) {
+      return Promise.reject(new Error('guid is required'));
+    }
+
+    const orgGuid = OrgStore.currentOrgGuid;
+
+    AppDispatcher.handleViewAction({
+      type: userActionTypes.CURRENT_USER_ROLES_FETCH,
+      userGuid,
+      orgGuid
+    });
+
+    return cfApi.fetchOrgUserRoles(orgGuid)
```

Comment:
This request is already happening in another action. Rather then calling the same thing twice, we should use that action to do what we need.